cmd/util.go

```diff
@@ -36,3 +36,18 @@ func ifFilesPassed(files []string) error {
 	}
 	return nil
 }
+
+// RemoveDuplicateFiles function will see input files and will remove duplicate entries
+func RemoveDuplicateFiles(inputFiles []string) []string {
+	files := map[string]bool{}
+	fileSet := []string{}
+
+	for v := range inputFiles {
+		if files[inputFiles[v]] == true {
```

Comment:
We are not checking in the `if` block, so we can make it `if !files[inputFiles[v]]`

Reply:
:+1: